infrastructure/code/code.go

```diff
@@ -116,10 +116,26 @@ func (sc *Scanner) SupportedCommands() []snyk.CommandName {
 }
 
 func (sc *Scanner) Scan(ctx context.Context, path string, folderPath string) (issues []snyk.Issue, err error) {
-	if !sc.isSastEnabled() {
+	sastResponse, err := sc.SnykApiClient.SastSettings()
```

Comment:
Isn't this a redundant call to the settings endpoint?

Reply:
Do we already have the sastSettings response somewhere? Why do you think its redundant?